### PR TITLE
add the comment of ConsistentHash

### PR DESCRIPTION
**Pull request by aliiohs**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cluster/loadbalance/consistent_hash.go [hunk 1]
```diff
@@ -85,22 +85,25 @@ func (lb *ConsistentHashLoadBalance) Select(invokers []protocol.Invoker, invocat
 	return selector.Select(invocation)
 }
 
-// Uint32Slice ...
+// Uint32Slice Slice of uint32
 type Uint32Slice []uint32
```
Comment (zouyx) [inline cluster/loadbalance/consistent_hash.go:89]: I think make this struct to private would be better.

File: cluster/loadbalance/consistent_hash.go [hunk 2]
```diff
@@ -53,16 +53,16 @@ func init() {
 	extension.SetLoadbalance(ConsistentHash, NewConsistentHashLoadBalance)
 }
 
-// ConsistentHashLoadBalance ...
+// ConsistentHashLoadBalance Implementation of load balancing: using consistent hashing
```
Comment (Patrick0308) [inline cluster/loadbalance/consistent_hash.go:56]: See https://golang.org/doc/effective_go.html#commentary, after function name, the first character of word don't need to be capitalized.

File: cluster/loadbalance/consistent_hash.go (hunk 1 above)
Comment (flycash) [reply cluster/loadbalance/consistent_hash.go:89]: please move this to gost. We already have similar structure...

Comment (hxmhlt) [reply cluster/loadbalance/consistent_hash.go:89]: > please move this to gost. We already have similar structure...

+1

Comment (aliiohs) [reply cluster/loadbalance/consistent_hash.go]: already replaced Uint32Slice  by gxsort.Uint32Slice

File: cluster/loadbalance/consistent_hash.go [hunk 3]
```diff
@@ -27,7 +27,9 @@ import (
 	"strconv"
 	"strings"
 )
-
+import (
+	gxsort "github.com/dubbogo/gost/sort"
+)
```
Comment (zouyx) [inline cluster/loadbalance/consistent_hash.go:32]:
```suggestion

import (
	gxsort "github.com/dubbogo/gost/sort"
)

```
i think should add a blank line before and after this block

**Pull request merged**